server/src/main/java/io/crate/replication/logical/MetadataTracker.java

```diff
@@ -224,12 +224,17 @@ private CompletableFuture<?> processSubscription(String subscriptionName) {
                     subscription.connectionInfo().user()
                 );
                 return client.execute(PublicationsStateAction.INSTANCE, request)
-                    .thenCompose(publicationsStateResp -> subscribeToNewRelations(
-                        subscriptionName,
-                        subscription,
-                        subscriberState,
-                        publicationsStateResp,
-                        replicationService));
+                    .thenCompose(publicationsStateResp -> {
+                        LOGGER.debug("subscriber got concreteIndices from a publisher: {}", publicationsStateResp.concreteIndices());
+                        LOGGER.debug("subscriber got concreteTemplates from a publisher: {}", publicationsStateResp.concreteTemplates());
+                        LOGGER.debug("subscriber got tables from a publisher: {}", publicationsStateResp.tables());
```

Comment:
To be removed, aded to check that ALTER PUBLICATION results are visible on a subscriber.

They are visible, log output before and after ALTER PUBLICATION:
```
[2022-04-21T13:54:37,020][DEBUG][i.c.r.l.MetadataTracker  ] [[cratedb[subscriber1][netty-worker][T#1]]] subscriber got concreteIndices from a publisher: [t1, t2]
[2022-04-21T13:54:37,020][DEBUG][i.c.r.l.MetadataTracker  ] [[cratedb[subscriber1][netty-worker][T#1]]] subscriber got concreteTemplates from a publisher: []
[2022-04-21T13:54:37,021][DEBUG][i.c.r.l.MetadataTracker  ] [[cratedb[subscriber1][netty-worker][T#1]]] subscriber got tables from a publisher: [doc.t1, doc.t2]
```

```
[2022-04-21T13:54:37,878][DEBUG][i.c.r.l.MetadataTracker  ] [[cratedb[subscriber1][netty-worker][T#1]]] subscriber got concreteIndices from a publisher: [t1]
[2022-04-21T13:54:37,878][DEBUG][i.c.r.l.MetadataTracker  ] [[cratedb[subscriber1][netty-worker][T#1]]] subscriber got concreteTemplates from a publisher: []
[2022-04-21T13:54:37,878][DEBUG][i.c.r.l.MetadataTracker  ] [[cratedb[subscriber1][netty-worker][T#1]]] subscriber got tables from a publisher: [doc.t1]
```